### ISSUE TITLE
Allow specifying credentials via parameters

### ISSUE THREAD

**Issue by MasterOdin** [Tomme/dbt-athena#38]
Via the [README#Credentials](https://github.com/Tomme/dbt-athena#credentials), this library only supports loading credentials through configured AWS CLI credentials. Would you support adding in the ability to configure the credentials (access key, secret key) as parameters to the connector? Looking at the underlying PyAthena library, it already supports it (see [here](https://github.com/laughingman7743/PyAthena/#passing-credentials-as-parameters)), so it would just be amending this library to allow parsing it in / passing it along I think. Happy to look to put up a PR for this, but want to confirm if it would be merged before committing to it.

**Comment by MasterOdin** [comment]
While I think environment variables would work as we execute each dbt rpc instance in its own process, it's a bit annoying to use for our implementation. We would end up probably still sticking those variables into the `profiles.yml` file (as we want to store all DB specific stuff for all adapters in the file), and then parse it out into environment variables during runtime. Of course, would then be nicer to just use the `profiles.yml` only to avoid that.

However does not sound like you're keen on supporting this usage (even if I provide a PR)?

**Comment by Dandandan** [comment]
> While I think environment variables would work as we execute each dbt rpc instance in its own process, it's a bit annoying to use for our implementation. We would end up probably still sticking those variables into the `profiles.yml` file (as we want to store all DB specific stuff for all adapters in the file), and then parse it out into environment variables during runtime. Of course, would then be nicer to just use the `profiles.yml` only to avoid that.

Makes sense to me 👍 just wanted to know whether the alternatives were clear.
